src/test/java/com/wikia/webdriver/common/core/elemnt/JavascriptActions.java

```diff
@@ -131,10 +132,11 @@ public void scrollToElement(WebElement element) {
   private int getOffset() {
     WikiBasePageObject wikiPage = new WikiBasePageObject();
     int offset = wikiPage.getNavigationBarOffsetFromTop();
-    if (wikiPage.isBannerNotificationContainerPresent()) {
-      int notificationsHeight = wikiPage.getBannerNotificationsHeight();
-      offset += notificationsHeight;
-    }
+    if (wikiPage.isBannerNotificationContainerPresent()) offset += wikiPage.getBannerNotificationsHeight();
+
+    Search searchComponent = new SearchResultsPage().getSearch();
+    if (searchComponent.isPresent()) offset += searchComponent.getHeight();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Missing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00121)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00122)